src/style.css

```diff
@@ -1 +1,85 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto , sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.main_header {
+  display: flex;
+  justify-content: space-between;
+  padding: auto 50px;
+  width: 100%;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  --focus_color: #00acdc;
+  margin: 0;
+  padding-inline-start: 0;
+  display: flex;
+  list-style: none;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.list__item:nth-child(n + 2) {
+  margin-left: 20px;
+}
+
+.link_list_item {
+  flex-basis: max-content;
+  word-wrap: normal;
+  --height: 60px;
+  text-align: center;
+  display: flex;
+  background-color: #fff;
+  height: var(--height);
+  line-height: var(--height);
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+}
+
+.link_list_item:hover {
+  color: var(--focus_color);
+}
+
+.logo_link {
+  position: relative;
+  margin-right: 50px;
+  height: 40px;
+  width: 40px;
+  top: 10px;
+}
+
+.is-active {
+  color: var(--focus_color);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 37px;
```

Comment:
If active will be another element it will looks like this one 
![2023-04-18_13h01_16](https://user-images.githubusercontent.com/61904995/232743422-d070a598-7351-4b61-83f5-0e607b72ade1.png)
But it should be 100% link width
![2023-04-18_13h02_36](https://user-images.githubusercontent.com/61904995/232743777-92ed415f-8891-46f4-a8fb-218558f221fe.png)